src/notify_me/script.py

```diff
@@ -0,0 +1,5 @@
+import requests
+
+
+def notify_me(message, config):
```

Comment:
What format will the config file be in? .json?

Reply:
probably, but a dictionary will be passed in here.